.github/workflows/main.yml

```diff
@@ -6,13 +6,7 @@ on:
   workflow_dispatch:
   push:
     branches: [main]
-    paths-ignore:
```

Comment:
Given that we now depend on the most recent gitsha from the pull request (PR) to include a container image, we should not hinder the continuous integration (CI) process from generating an image in cases where only a documentation change is made.

Reply:
Can you leave an inline comment. Maybe something like `OPTIMIZE: We generate new images even on non src code changes, but this cost is okay for now`